src/infi/clickhouse_orm/query.py

```diff
@@ -103,6 +103,21 @@ def to_sql(self, model_cls, field_name, value):
         return 'NOT (%s)' % self._base_operator.to_sql(model_cls, field_name, value)
 
 
+class BetweenOperator(Operator):
+
```

Comment:
Would be great to have a docstring here in newly introduced class

Reply:
Yes, I'll do it soon

Reply:
Done!